test_async.py

```
@@ -0,0 +1,108 @@
+sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)))
+
+async def test_async_basic():
+    """Test basic async database operations."""
+    print("Testing async basic operations...")
+    
+    try:
+        from async_records import AsyncDatabase
+        
+        # Note: For this test we'll use a regular sqlite URL 
+        # In practice, you'd want to use aiosqlite for true async SQLite
+        async with AsyncDatabase('sqlite:///memory:') as db:
```

Ruff (0.14.0)

Local variable `db` is assigned to but never used

Remove assignment to unused variable `db`

(F841)